src/pages/home/report/ReportActionsView.tsx

```diff
@@ -49,34 +52,63 @@ type ReportActionsViewProps = ReportActionsViewOnyxProps & {
 
     /** The report actions are loading newer data */
     isLoadingNewerReportActions?: boolean;
+
+    /** Whether the report is ready for comment linking */
+    isReadyForCommentLinking?: boolean;
 };
 
+const DIFF_BETWEEN_SCREEN_HEIGHT_AND_LIST = 120;
+const SPACER = 16;
+
+let listOldID = Math.round(Math.random() * 100);
```

Comment:
@perunt This is against React rules, why did you decide to keep state outside of React component? 🤔

Reply:
As far as I remember, it helped me fix a bug while implementing the scrolling functionality before adding pagination. Most likely, it was related to avoiding changes to this property with each navigation between the same screens or something similar—I can't fully recall this part